### PR TITLE
feat(NF-1877): Implemented TC103 for Owner/ContOwner/SecondAddresee

### PR DESCRIPTION
**Pull request by saivarunthupakula-sureify**
# Description
Implemented TC103 for Owner , Contingent owner and secondary addressee.

# What's new?

# Jira Link(s)
https://sureify.atlassian.net/browse/NF-1877

# Demo / Screen Shot (as applicable)
[Screencast from 31-05-24 05:58:02 PM IST.webm](https://github.com/sureifylabs/acquire-tc103/assets/133852700/5532633a-6b8b-48ad-8129-5f04a184c89c)

# PR Author Checklist
- [ ] Unit tests added, already exist, or are not applicable
- [x] Added necessary label(s)
- [x] Assigned corresponding reviewer(s)
- [ ] Updated README / CHANGELOG (if applicable)

## Author Notes
- Add label for least one of:
    - Ready for Review
    - WIP
    - don't merge
- If the PR is not yet ready to be assigned to reviewers [Create a Draft PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft)
- If a reviewer comments on something that could have been caught by a lint rule, either add that rule to the linter, or create a Jira ticket to add it.

## Reviewer Notes
If any PR doesn't meet the above standards, please feel free to reject the PR. :evil-laugh:

### CODE REVIEW

**Review by Naveen-Sureify — APPROVED** [review]

File: packages/xml/src/templates/partials_nw/PartyOwner_NW.ejs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  </Person>
+  <GovtIDTC tc="1">SSN</GovtIDTC>
+  <GovtID><%=acord?.PO?.SSN?.replace(/[()\-]/g, '')%></GovtID>
+  <Address>
+    <AddressTypeCode tc="1">Residence</AddressTypeCode>
+    <Line1><%=acord?.PO?.Address%></Line1>
+    <Line2><%=acord?.PO?.Address%></Line2>
+    <City><%=acord?.PO?.City%></City>
+    <AddressState><%=acord?.PO?.State%></AddressState>
+    <AddressStateTC tc="<%= await lookups.lookup(types.OLI_LU.DRIVERSLICENSESTATENW, {value: acord?.PO.State}).tc%>">
```
Comment (manyumanchala-sureify) [inline packages/xml/src/templates/partials_nw/PartyOwner_NW.ejs:20]: Missed optional accessing here
```suggestion
    <AddressStateTC tc="<%= await lookups.lookup(types.OLI_LU.DRIVERSLICENSESTATENW, {value: acord?.PO?.State}).tc%>">
```

File: packages/xml/src/templates/partials_nw/PartyOwner_NW.ejs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    <AddrLine><%=acord?.PO?.EmailAddress%></AddrLine>
+  </EmailAddress>
+  <Phone>
+    <AreaCode><%=acord?.PO?.PhoneNumber?.substring(0,5)?.replace(/[()\-]/g, '')%></AreaCode>
+    <DialNumber><%=acord?.PO?.PhoneNumber?.substring(6)?.replace(/[()\-]/g, '')%></DialNumber>
+    <PhoneTypeCode tc="<%= await lookups.lookup(types.OLI_LU.PHONETYPE, {value: acord?.PO?.PhoneType}).tc%>"><%=acord?.PO?.PhoneType%></PhoneTypeCode>
+  </Phone>
+  <%_ } _%>
+  <Risk>
+    <OLifEExtension><%_ if(acord?.PI.IsPIOwner === "No"){ _%>
```
Comment (manyumanchala-sureify) [inline packages/xml/src/templates/partials_nw/PartyOwner_NW.ejs:35]:
```suggestion
    <OLifEExtension><%_ if(acord?.PI?.IsPIOwner === "No"){ _%>
```

File: packages/xml/src/templates/partials_nw/PartyOwner_NW.ejs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      <BackupInd><%=acord?.PO?.IRSnotified%></BackupInd><%}%><%_ if(acord?.PI.IsPIOwner === "Yes"){ _%>
+      <BackupInd><%=acord?.PI?.IRSnotified%></BackupInd><%_ } _%>
```
Comment (manyumanchala-sureify) [inline packages/xml/src/templates/partials_nw/PartyOwner_NW.ejs:37]: I think we can use else if here just for better conditioning

File: packages/xml/src/templates/partials_nw/PartyOwner_NW.ejs (hunk 1 above)
Comment (saivarunthupakula-sureify) [reply packages/xml/src/templates/partials_nw/PartyOwner_NW.ejs]: Added optional accessing.

File: packages/xml/src/templates/partials_nw/PartyOwner_NW.ejs (hunk 3 above)
Comment (saivarunthupakula-sureify) [reply packages/xml/src/templates/partials_nw/PartyOwner_NW.ejs:37]: Replaced if with else if

**Review by manyumanchala-sureify — APPROVED** [review]
LGTM :100:

**Pull request merged**